server/api/reports.py

```diff
@@ -775,83 +786,110 @@ def sampled_batch_rows(election: Election, jurisdiction: Jurisdiction = None):
         )
     }
 
-    rows.append(
-        [
-            "Jurisdiction Name",
-            "Batch Name",
-            "Ticket Numbers",
-            "Audited?",
-            "Audit Results",
-            "Reported Results",
-            "Change in Results",
-            "Change in Margin",
-            "Last Edited By",
-        ]
+    contests = list(
+        election.contests if jurisdiction is None else jurisdiction.contests
     )
+
+    ticket_number_columns = []
+    result_columns = []
+    for contest in contests:
+        ticket_number_columns += [f"Ticket Numbers: {contest.name}"]
+        result_columns += [
+            f"Audit Results: {contest.name}",
+            f"Reported Results: {contest.name}",
+            f"Change in Results: {contest.name}",
+            f"Change in Margin: {contest.name}",
+        ]
+    column_headers = [
+        "Jurisdiction Name",
+        "Batch Name",
+        *ticket_number_columns,
+        "Audited?",
+        *result_columns,
+        "Last Edited By",
+    ]
+    rows.append(column_headers)
```

Comment:
I'm mimicking the table structure used to represent multi-contest non-batch audits. Here are some sample reports to make the structure clearer:

- [1-round-audit-report.csv](https://github.com/votingworks/arlo/files/14377962/1-round-audit-report.csv)
- [2-round-audit-report.csv](https://github.com/votingworks/arlo/files/14377963/2-round-audit-report.csv)

You can infer whether a batch's results were actually considered by the audit math by looking at the ticket numbers.